collect_app/src/main/res/color/color_on_surface_medium_emphasis.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would tend to leave a comment with some of the context at https://github.com/opendatakit/collect/pull/3207/commits/a81dd497e98d90fbcc647299d9c1921859c1fde3#r302561112? I might be an over-commenter, though.

Reply:
Do you mean a comment in the code itself or as part of the PR? I actually feel like I'm probably under commenting so good to hear feedback when you think things need some comments - always working with teams all in one place is a good way to think nothing needs comments.

Reply:
I mean as an XML comment. They're allowed there, right?

Reply:
Yeah totally. I can look at adding some comment to this!